src/WebApp/Pages/Questions.cshtml

```diff
@@ -0,0 +1,65 @@
+﻿@page "/{category}/questions"
+@model Codidact.Core.WebApp.Pages.Posts.QuestionsModel
+@using Codidact.Core.Application.Questions.Queries
+
+@{
+    ViewData["Title"] = "Questions";
+    var query = Model.Query;
+}
+
+<partial name="_ValidationSummary" />
+
+
+<div class="grid has-margin-4">
+    <div class="grid--cell is-flexible">
+        <h1 class="has-margin-0">Q&amp;A posts</h1>
```

Comment:
Q&A posts sounds strange. Here should be the category name and/or the post type. So something like "Questions" or "Main Questions".

Reply:
I fixed this by adding to the query result the category but ideally I should have an area/component/whatever that does this separately from the questions query like some community data.

Also added a short explanation below the category name.

```
    <div class="grid--cell is-flexible">
        <h1 class="has-margin-0">@Model.Result.Category.DisplayName Questions</h1>
        <h3>@Model.Result.Category.ShortExplanation</h3>
    </div>
```